src/mcpbr/log_formatter.py

```
@@ -504,7 +506,7 @@ def create_formatter(
     log_file: TextIO | None = None
     if log_file_path:
         log_file_path.parent.mkdir(parents=True, exist_ok=True)
-        log_file = open(log_file_path, "w")
+        log_file = open(log_file_path, "w")  # noqa: SIM115 - caller closes the file handle
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `SIM115`)

Remove unused `noqa` directive

(RUF100)